src/app/shared/forms/utils/form-utils.ts

```diff
@@ -35,3 +35,34 @@ export function ariaDescribedByIds(id: string, showError: boolean, customDescrip
 
   return describedByIds || undefined;
 }
+
+/**
+ * Set focus on the first invalid field in a form.
+ *
+ * @param form The form group to check for invalid fields.
+ */
+export function focusFirstInvalidField(form: FormGroup) {
```

Comment:
Silke said we shouldn't use form-utils anymore and use the forms.service instead, so the function should be moved there